web-frontend/src/pages/TransferCoins/SendCoins/Advanced.tsx

```diff
@@ -166,7 +166,7 @@ export default function Advanced(props: AdvancedProps) {
               placeholder={Intl.t('send-coins.custom-fees')}
               name="fees"
               variant="MAS"
-              value={feesField}
+              value={(customFees && currentFees) || feesField}
```

Comment:
what is the logic here? if customFees === bool then currentFees else feesField?

or 

if (customFees === bool and currentFees !== '') then currentFees else feesField

this syntax `customFees && currentFees) || feesField` is not really clear to me.

Reply:
I pushed a modification of the syntax. Indeed : 
```customFees === bool then currentFees else feesField?```